### PR TITLE
Dev to Main 

### PR DESCRIPTION
**Pull request by kiasha10**
## Description
Development branch into Main

## Checklist
Please review and check the following before submitting this pull request:
- [x] The code follows the project's coding standards.
- [ ] 100% Unit test coverage for View Models

### CODE REVIEW

**Review by AShozi — APPROVED** [review]

**Review by ksookdhew — APPROVED** [review]

File: BlitzBoard/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "images" : [
+    {
+      "filename" : "iTunesArtwork@2x.png",
```
Comment (kieran-woodrow) [inline BlitzBoard/Assets.xcassets/AppIcon.appiconset/Contents.json:4]: please name this asset better

File: BlitzBoard/Assets.xcassets/TeamBadgePlaceholder.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "pngtree-internet-ranking-icon-for-your-project-png-image_1571193.jpg",
```
Comment (kieran-woodrow) [inline BlitzBoard/Assets.xcassets/TeamBadgePlaceholder.imageset/Contents.json:4]: and here

File: BlitzBoard/Features/FixturesScreenViewController.swift [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+//
+
+import UIKit
+
+class FixturesScreenViewController: UIViewController {
+    
+    // MARK: IBOutlets
+    
+    @IBOutlet private weak var tableView: UITableView!
+    @IBOutlet private weak  var spinner: UIActivityIndicatorView!
```
Comment (kieran-woodrow) [inline BlitzBoard/Features/FixturesScreenViewController.swift:14]: extra space here

File: BlitzBoard/Features/FixturesScreenViewController.swift [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    }
+}
+
+// MARK: - Extensions
+
+extension FixturesScreenViewController: UITableViewDataSource, UITableViewDelegate {
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        viewModel.numberOfGames
+
```
Comment (kieran-woodrow) [inline BlitzBoard/Features/FixturesScreenViewController.swift:43]: extra new line here

File: BlitzBoard/Features/FixturesScreenViewController.swift [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+extension FixturesScreenViewController: ViewModelDelegate {
+    
+    func reloadView() {
+            self.tableView.reloadData()
+            self.spinner.isHidden = true
+    }
+    
+    func show(error: String) {
+
```
Comment (kieran-woodrow) [inline BlitzBoard/Features/FixturesScreenViewController.swift:70]: empty function? Is this meant to be empty?

File: BlitzBoard/Features/HomeScreenViewModel.swift [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    func reloadView()
+    func show(error: String)
+}
+
+class HomeScreenViewModel {
+    
+    // MARK: Variables
+    
+    var leagueTables: [LeagueTableModel]
+
```
Comment (kieran-woodrow) [inline BlitzBoard/Features/HomeScreenViewModel.swift:19]: can remove this new line

File: BlitzBoard/HomePage/Repository/HomePageRepository.swift [hunk 7]
```diff
@@ -0,0 +1,8 @@
+//
+//  HomePageRepository.swift
```
Comment (kieran-woodrow) [inline BlitzBoard/HomePage/Repository/HomePageRepository.swift:2]: please don't commit empty files

File: BlitzBoard/HomePage/Repository/HomePageModel.swift [hunk 8]
```diff
@@ -0,0 +1,8 @@
+//
+//  HomePageModel.swift
+//  BlitzBoard
+//
```
Comment (kieran-woodrow) [inline BlitzBoard/HomePage/Repository/HomePageModel.swift:4]: please don't commit empty files

File: BlitzBoard/Repository/LeagueTableModel.swift [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+//  BlitzBoard
+//
+//  Created by Kiasha Rangasamy on 2024/04/11.
+//
+
+import Foundation
+
+// MARK: - League Table Model
+struct LeagueTableModel: Codable {
+    var countryName: CountryName
```
Comment (kieran-woodrow) [inline BlitzBoard/Repository/LeagueTableModel.swift:12]: are all of these being used?

**Review by kieran-woodrow — APPROVED** [review]
Please address comments i a separate pr

**Comment by kiasha10** [comment]
Forced merge by Lloyd

**Pull request merged**